src/domains/Feedback/Rating/Rating.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import {
+    Typography,
+    Grid,
+    Button,
+    TextField,
+} from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+import { makeStyles } from '@material-ui/core/styles';
+import type { RatingForm } from 'prytaneum-typings';
+
+import LoadingButton from 'components/LoadingButton';
+import useEndpoint from 'hooks/useEndpoint';
+
+import { rateTownhall } from '../api';
+
+interface DefaultProps {
+    townhallId: string;
+}
```

Comment:
I didn't notice this last time, but shouldn't townhallId just be a regular Prop?

Reply:
Yeah, it probably should. It had more props before but I ended up removing them and just left it that way. I'll update that right now.